src/routes/user.route.ts

```diff
@@ -36,6 +37,7 @@ router.post("/register", async (req: Request, res: Response) => {
     const user = await createUser(req.body);
 
     logger.info("User created: " + user.id);
+    sendConfirmationEmail(user.email);
```

Comment:
What happens if `createUser` function breaks? maybe prisma fails for some reason. You need to handle that case when something unexpected happens.

Reply:
This code is in a `try/catch` block. My understanding is that if that function (or any other function in the `try` block) fails, code in the `catch` block will be executed. It should return `Internal server error`.

Reply:
Ahh, didn't see try and catch block. All good.